### PR TITLE
constraints: make it work with spec-stats

### PR DESCRIPTION
**Pull request by fippo**
makes the "connected to" in the constraints sample work with spec-stats

@henbos can you please take a look?

### CODE REVIEW

File: src/content/peerconnection/constraints/js/main.js [hunk 1]
```diff
@@ -249,20 +249,31 @@ setInterval(function() {
...
-            report.type === 'googCandidatePair' &&
-            report.googActiveConnection === 'true') {
-          activeCandidatePair = report;
+        if (report.type === 'transport') { 
+          activeCandidatePair = results.get(report.selectedCandidatePairId); 
         }
       });
+      if (!activeCandidatePair) {
+        results.forEach(function(report) {
+          if (report.type === 'candidate-pair' && report.selected ||
```
Comment (fippo) [inline src/content/peerconnection/constraints/js/main.js]: using candidate-pair here fix things in Firefox too

Comment (henbos) [reply src/content/peerconnection/constraints/js/main.js]: Can you merge these two searches and add a comment that transport.selectedCandidatePairId is spec way and candidatePair.selected is Firefox way?

**Review by henbos — APPROVED** [review]

File: src/content/peerconnection/constraints/js/main.js (hunk 1 above)
Comment (fippo) [reply src/content/peerconnection/constraints/js/main.js]: comments added. I think it is easier to read and understand what is the legacy fallback when the forEach's are separated

Comment (fippo) [reply src/content/peerconnection/constraints/js/main.js]: also added a comment about ipAddress and portNumber below.

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**